From 3e46bdfe8e0262c3dd8858c0b4475a9721e1fc00 Mon Sep 17 00:00:00 2001
From: "Chen, Yu" <yu.y.chen@intel.com>
Date: Tue, 11 Jun 2019 14:23:42 +0800
Subject: [PATCH 2/2] [WA]Don't build the sepolicy_freeze_test

this module will check the sepolicy changes by diff
the source code with the prebuilts, it'll report error if
find the sepolicy changes.

Now, below patch in google_diff makes change on sepolicy,
https://android.intel.com/#/c/667910/5

Change-Id: Idbfef213422a875a1645dd2062d9100bcebb5193
Tracked-On:
Signed-off-by: Chen, Yu <yu.y.chen@intel.com>
---
 Android.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 361cc30e..449c21d3 100644
--- a/Android.mk
+++ b/Android.mk
@@ -244,7 +244,6 @@ endif

 ifneq ($(PLATFORM_SEPOLICY_VERSION),$(TOT_SEPOLICY_VERSION))
 LOCAL_REQUIRED_MODULES += \
-    sepolicy_freeze_test \

 endif # ($(PLATFORM_SEPOLICY_VERSION),$(TOT_SEPOLICY_VERSION))

--
2.21.0
